Sources/Clappr/Classes/Base/Playback.swift

```diff
@@ -96,7 +96,9 @@ open class Playback: UIBaseObject, Plugin {
         }
 
         #if os(tvOS)
-        play()
+        DispatchQueue.main.asyncAfter(deadline: .now()) {
```

Comment:
Hey,
We are not using `[weak self]` here, maybe this cause a memory leak problem if someone tries to play and dismiss the player right away.